### PR TITLE
OCLOMRS-498: Remove the “Other” Option from Bulk Add Concepts page

### PR DESCRIPTION
**Pull request by judeinno**
# JIRA TICKET NAME:
[Remove the “Other” Option from Bulk Add Concepts page](https://issues.openmrs.org/browse/OCLOMRS-498)

# Summary:
Remove the Other radio button on the Bulk Add Concepts page it will be re-implemented in post MVP.

### CODE REVIEW

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 1]
```diff
@@ -58,23 +57,11 @@ export class AddBulkConcepts extends Component {
...
   };
 
-  otherSourceClick = () => {
-    this.setState({ otherSelected: true });
-    this.props.fetchConceptSources();
-    this.reset();
-  }
-
-  handleSourceSelect = (source) => {
-    this.sourceUrl = source.url.substring(1);
```
Comment (dkayiwa) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: How is this related to removing other?

File: src/components/bulkConcepts/addBulkConcepts.jsx [hunk 2]
```diff
@@ -127,17 +114,15 @@ export class AddBulkConcepts extends Component {
...
-    const { conceptIds, openResultModal, otherSelected } = this.state;
+    const { conceptIds, openResultModal } = this.state;
     const {
       match: {
         params: {
           type, typeName, collectionName, language,
         },
       },
       sourceConcepts,
-      conceptSources,
```
Comment (dkayiwa) [inline src/components/bulkConcepts/addBulkConcepts.jsx]: How is this related to removing other?

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 1 above)
Comment (judeinno) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: This was being used by the other search to search for the sources.

File: src/components/bulkConcepts/addBulkConcepts.jsx (hunk 2 above)
Comment (judeinno) [reply src/components/bulkConcepts/addBulkConcepts.jsx]: This was used by the others search field to filter the source names by the input data provided. This is shown on line 228 that I had to clear.

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx [hunk 3]
```diff
@@ -74,27 +74,6 @@ describe('Add Bulk Concepts', () => {
     expect(wrapper.find('Loader')).toHaveLength(1);
   });
 
-  it('should render a loader while fetching Concepts Source', () => {
-    const properties = {
-      fetchSourceConcepts: jest.fn(),
-      addExistingBulkConcepts: jest.fn(),
-      fetchConceptSources: jest.fn(),
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/addBulkConcepts.test.jsx]: What happens if we leave this test?

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx [hunk 4]
```diff
@@ -207,39 +186,6 @@ describe('Add Bulk Concepts', () => {
...
-    const wrapper = mount(<MemoryRouter>
-      <Provider store={store}>
-        <AddBulkConcepts {...props} />
-      </Provider>
-    </MemoryRouter>);
-    const other = wrapper.find('#otherSourcesOption');
-    other.simulate('click');
-    const input = wrapper.find('#sourceSearch');
-    input.simulate('change', { target: { value: 'testSource' } });
-    input.simulate('keydown', { key: 'ArrowDown' });
```
Comment (dkayiwa) [inline src/tests/bulkConcepts/addBulkConcepts.test.jsx]: What happens if we leave this test?

Comment (judeinno) [reply src/tests/bulkConcepts/addBulkConcepts.test.jsx]: It will fail because its searching through the component to find an Id/node `#otherSourcesOption`that does not exist hence it will throw an error saying. `This method is only meant to be run on single node. 0 found instead.`

File: src/tests/bulkConcepts/addBulkConcepts.test.jsx (hunk 3 above)
Comment (judeinno) [reply src/tests/bulkConcepts/addBulkConcepts.test.jsx]: It will fail because it will try to find a loader of length 1 but instead find length 0 because since we are searching from non-CIEL concepts it will search through the other sources and since that field no longer exists the test will fail.
